selftests/unit/test_job.py

```diff
@@ -329,7 +329,6 @@ def test_job_make_test_suite_resolver(self):
     def test_job_get_failed_tests(self):
```

Comment:
Not sure if I understood why `run.dry_run.enabled` makes no sense here?

IIUC, this test is testing failed test counts, and should not be changed or converted to dry_run. a /bin/true + /bin/false should continue to work and having only 1 failed.

If we need to test the dry-run, this could be in a different test.

Reply:
That is what the test is doing, checking how many tests failed. `run.dry_run.enabled` here will result in 2 canceled, not one pass and one canceled.

Reply:
As @willianrampazzo said, before the 442d069629dc3efbe9ed17aa64b5206badfe025f the `run.dry_run.enabled` wasn't changing the behavior of nrunner and the test was running like a standard test. But now without this commit the job will be run as `dry-run` and cancel all tests. That is why we have to remove `run.dry_run.enabled` or we can change it to `False` but that is the default value.

Reply:
Nevermind, I got it.